frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -59,7 +59,15 @@
     <p class='field' data-hook='coupon_code'>
       <%= form.label :coupon_code %><br />
       <%= form.text_field :coupon_code %>
+      <button type="button" class="button" id="coupon-code-apply-button">
+        <%= Spree.t(:apply_code) %>
+      </button>
     </p>
+    <div id='coupon_status'
```

Comment:
I believe this should be inside the above `<p>` tag to match the existing behaviour.